src/App.java

```diff
@@ -1,45 +1,17 @@
 import java.io.FileReader;
 import java.io.PrintWriter;
 import java.util.Arrays;
+import java.util.HashMap;
 
 import java.util.Scanner;
 import java.util.ArrayList;
 public class App {
-    final ArrayList<String> OPERATORS = new ArrayList<String>(Arrays.asList(
```

Comment:
Good moving these variables into the tokenLexemeParser() to avoid global variables